mix.exs

```diff
@@ -18,41 +18,38 @@ defmodule Barrel.Mixfile do
   def application do
     [
       mod: {:barrel_app, []},
-      applications:
+      applications: 
       [
-        :barrel_ex_logger,
         :kernel,
         :stdlib,
         :crypto,
-        :sasl,
         :asn1,
         :public_key,
         :ssl,
         :os_mon,
         :inets,
         :p1_utils,
-        :gproc,
-        :ibrowse,
+        :fast_yaml,
         :jsx,
+        :ibrowse,
+        :snappy,
         :ucol,
         :oauth,
-        :fast_yaml,
-        :ranch,
-        :cowboy,
+        :hackney,
         :mochiweb,
         :mochicow,
+        :cowlib,
+        :ranch,
+        :cowboy,
         :hooks,
-        :econfig,
-        :hackney,
-        :exometer_core,
-        :snappy
+        :exometer_core
       ]
     ]
   end
 
   defp deps do
     [
-      {:barrel_ex_logger, git: "https://github.com/barrel-db/barrel_ex_logger.git"},
+      {:lager, "~> 3.0.2"},

```

Comment:
do you really need it with elixir? The real question to keep lager or not in elixir is more about to use the common pattern for elixir apps:

what do they use in Elixir world to store their logs in files or forward them to something like kibana?